app/src/main/java/com/example/concordia_campus_guide/ClassConstants.java

```diff
@@ -16,8 +12,6 @@
 import java.util.List;
 
 public class  ClassConstants {
-    public static final String FINE_LOCATION = Manifest.permission.ACCESS_FINE_LOCATION;
-    public static final int LOCATION_PERMISSION_REQUEST_CODE = 1234;
     public static boolean MY_LOCATION_PERMISSION_GRANTED = false;
     public static final String DISABILITY_BUTTON = "disability_button";
     public static final String CALENDAR_INTEGRATION_BUTTON = "calendar_integration_button";
```

Comment:
Where is this used ?

Reply:
app/src/main/java/com/example/concordia_campus_guide/Helper/CurrentLocationPermissionRequest.java

Reply:
Then, maybe we can just keep it in there since it is not used anywhere else too